Gemfile.lock

```diff
@@ -91,6 +93,10 @@ GEM
     erubi (1.11.0)
     globalid (1.0.0)
       activesupport (>= 5.0)
+    haml (6.0.10)
```

Comment:
Haml is not the best language for learning. It can get confusing quite quickly. ERB will do just fine, but it's just an opinion

Reply:
Yeah, I agree, but I'm simply following the requirements on Techmap for this